changelog.d/56.misc

```diff
@@ -0,0 +1 @@
+Temporarily revert #7606.
```

Comment:
```suggestion
Temporarily revert commit a3fbc23.
```

I think the commit hash is more useful here as #7606 doesn't exist on this repo.

Reply:
Fair nuff. Not that we use the changelog for anything on this repo (we should probably take that step out of the CI at some point?).